ricecooker/classes/files.py

```diff
@@ -559,8 +559,8 @@ def get_preset(self):
 
     def process_file(self):
         self.filename = super(AudioFile, self).process_file()
-        self.duration = extract_duration_of_media(self.filename)
-
+        self.duration = extract_duration_of_media(self.path)
```

Comment:
I think we actually want to use: `config.get_storage_path(self.filename)` here, instead of `self.path` - for a file downloaded from the Internet, `self.path` will be a URL.

See here for example: https://github.com/learningequality/ricecooker/blob/33ca5c2cfdbae1057e2880f17790e55da3f79048/ricecooker/classes/files.py#L1193

Reply:
self.path is working fine i do not think that we should change this.

Reply:
If we are using this with get_storage_path(filename ) then we need extra check in the function because no all test got filename, some test where file does not exists will fail.